mo_net/samples/cbow.py

```
@@ -469,54 +541,44 @@ def infer(model_path: Path):
-            else:
-                similarity = np.dot(word1_embedding, word2_embedding) / (n1 * n2)
+    token_ids = [vocab[token] for token in tokens]
+    context = (
+        [vocab.unknown_token_id] * (context_width - len(token_ids)) + token_ids
+        if len(token_ids) < context_width
+        else token_ids[-context_width:]
+    )
+    predicted_tokens = []
 
-            click.echo(f"Similarity between '{word1}' and '{word2}': {similarity:.4f}")
+    for i in range(predict_tokens):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)